errai-bom/pom.xml

```diff
@@ -144,8 +147,8 @@
     <version.org.powermock>2.0.7</version.org.powermock>
     <version.org.quartz-scheduler>2.3.2 </version.org.quartz-scheduler>
     <version.org.slf4j>1.7.26</version.org.slf4j>
-    <version.org.wildfly>23.0.1.Final</version.org.wildfly>
-    <version.org.wildfly.arquillian>3.0.1.Final</version.org.wildfly.arquillian>
+    <version.org.wildfly>23.0.2.Final</version.org.wildfly>
+    <version.org.wildfly.arquillian>2.2.0.Final</version.org.wildfly.arquillian>
```

Comment:
this also should not be reverted from 3.0.1.Final as Wildfly may not work properly with older version. WF uses https://github.com/wildfly/wildfly/blob/23.x/pom.xml#L470 that version.